nodemongobearer/controller/employees.js

```diff
@@ -0,0 +1,89 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var multer = require('multer'); 
+var upload = multer(); 
+var app = require('express')();
+app.use(bodyParser.json()); 
+app.use(bodyParser.urlencoded({ extended: true })); 
+var router = express.Router();
+var request = require('request');
+var http = require('http');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/test');
+var db = mongoose.connection;
+var Schema = mongoose.Schema;
+var passport = require('passport');
+
+
+var userSchema = new Schema({
+  u_id: String
+  
+});
+var User = mongoose.model('User', userSchema,'User');
+
+var BearerStrategy = require('passport-http-bearer').Strategy; 
+var employeeModel = require('../model/employeeModel.js');
+module.exports = router;
+
+passport.use(new BearerStrategy(
```

Comment:
@psinha20 Write the code 'new BearerStrategy(' into separate file and import that file into the main(current) file.It makes
code more readable.

Reply:
@psinha20 Where is your package.json file?